src/main/java/com/theelixrlabs/healthcare/advice/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,44 @@
+package com.theelixrlabs.healthcare.advice;
+
+import com.theelixrlabs.healthcare.response.FailureResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Global exception handler to handle exceptions thrown by controller and service layers.
+ * Validates input parameters based on the specified annotations.
+ */
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+    /**
+     * Handles MethodArgumentNotValidException thrown when input parameters fail validation.
+     * @param methodArgumentNotValidException   The exception instance thrown during validation.
+     * @return ResponseEntity with FailureResponse containing validation error message and HTTP status BAD_REQUEST.
+     */
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<FailureResponse> handleInvalidArgument(MethodArgumentNotValidException methodArgumentNotValidException){
```

Comment:
Do proper reformatting 
spaces befor braces operators should be added and please follow the same for all